ci/Jenkinsfile.upload_release_android

```diff
@@ -55,7 +55,7 @@ timeout(90) {
         stage('Tag Build') {
           withCredentials([[
             $class: 'UsernamePasswordMultiBinding',
-            credentialsId: 'jenkins-status-im',
```

Comment:
I believe there is one more place in this file to be changed

Reply:
actually, that step shouldn't be there, the `build_no.sh` script does that.

Reply:
removed it

Reply:
then this step is also in `upload_ios`

Reply:
oh right, fixed